### ISSUE TITLE
On install: Cannot use import outside of module

### ISSUE THREAD

**Issue by peteromano** [storybook-vue/storybook-nuxt#55]
```
pnpm dlx nuxi@latest init my-app
cd my-app
pnpm dlx storybook-nuxt init
```

![image](https://github.com/storybook-vue/storybook-nuxt/assets/392387/7dcccbd1-9edb-4d0d-b0f9-f05cdec276ab)

**Comment by chakAs3** [comment]
@peteromano  i published new version cli, simpler please use it

**Comment by peteromano** [comment]
![image](https://github.com/storybook-vue/storybook-nuxt/assets/392387/fd9c9e06-66e6-44a6-b82c-39a01d037d79)

Hey, still has the issue, this is with `npx storybook-nuxt@0.0.5-alpha.2 init`

**Comment by peteromano** [comment]
![image](https://github.com/storybook-vue/storybook-nuxt/assets/392387/8a9cd526-9500-4fb5-9c3c-abd744dacc2f)

**Comment by peteromano** [comment]
Seems like this got fixed in 0.0.6-alpha.2
